qa/suites/fs/top/objectstore

```diff
@@ -0,0 +1 @@
+../../../objectstore
```

Comment:
Change this to copy `fs:volumes/objectstore`. No reason to test with more than 1 objectstore backend.